site/content/en/docs/start/_index.md

```diff
@@ -139,15 +139,15 @@ Click on the buttons that describe your target platform. For other architectures
 {{% quiz_instruction id="/Linux/x86-64/Stable/Binary download" %}}
 ```shell
 curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
-sudo install minikube-linux-amd64 /usr/local/bin/minikube
+sudo install minikube-linux-amd64 /usr/local/bin/minikube && rm -rf minikube-linux-amd64
```

Comment:
No need for `-rf`

Reply:
You are right. Changed. I have not suggested the change for other distributions because even though it must work equal, there could be changes or produce bugs on the installation and i am not able to test before commit.

Reply:
If you are going to do this, it should be the same also for other arch such as `minikube-linux-arm64`

Reply:
I have uploaded a commit to make the change on other linux distros.

Reply:
Thanks for all the reviewing (first time contributing to public repository). I have noticed that I need the #lgtm label